plugins/modules/firewall_rules.py

```diff
@@ -0,0 +1,448 @@
+        default: any
+        type: str
+    source:
+        description:
+          - Specifies the source configuration.
+        type: dict
+        suboptions:
+          address:
+            description:
+              - The IP address of the source.
+            default: any
+            type: str
+          network:
+            description:
+              - The network of the source.
+            default: any
+            type: str
+          port:
+            description:
+              - The port of the source.
+            default: any
+            type: str
+          invert:
+            description:
+              - Inverts the match logic.
+            default: false
+            type: bool
+    destination:
+        description:
+          - Specifies the source configuration.
+        type: dict
+        suboptions:
+          address:
+            description:
+              - The IP address of the source.
+            type: str
+            default: any
+          network:
+            description:
+              - The network of the source.
+            type: str
+            default: any
+          port:
+            description:
+              - The port of the source.
+            type: str
+            default: any
+          invert:
+            description:
+              - Inverts the match logic.
+            default: false
+            type: bool
+    log:
+        description: |
+          "Log packets that are handled by this rule. Hint: the firewall has limited local log space. Don't turn on logging for everything.
+          If you want to do a lot of logging, consider using a remote syslog server."
+        required: false
+        default: false
+        type: bool
+    category:
+        description: You may enter or select a category here to group firewall rules
+        required: false
+        type: str
+    description:
+        description: Description for the rule.
+        required: false
+        type: str
+    state:
+        description: Weather rule should be added or removed.
+        required: false
+        type: str
+        default: present
+        choices: [present, absent]
+author:
+    - Fabio Bertagna (@dongiovanni83)
+    - Kilian Soltermann (@killuuuhh)
+'''
+
+EXAMPLES = r'''
+- name: Block SSH in LAN Network
+  puzzle.opnsense.firewall_rules:
+    interface: lan
+    source:
+    destination:
+        port: 22
+    action: block
+
+- name: Allow all access from RFC1918 networks to this host
+  puzzle.opnsense.firewall_rules:
+    interface: lan
+    action: pass
+    source:
+        ip: 192.168.0.0/16
+    destination:
+'''
+
+RETURN = '''
+opnsense_configure_output:
+    description: A List of the executed OPNsense configure function along with their respective stdout, stderr and rc
+    returned: always
+    type: list
+    sample:
+      - function: "system_cron_configure"
+        params: []
+        rc: 0
+        stderr: ""
+        stderr_lines: []
+        stdout: ""
```

Comment:
This might change if the configure function were executed with `verbose=true`, same for `filter_configure`

Reply:
I have added the verbose output of the `system_cron_configure` function in the RETURN doc. 👍🏼